app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,33 @@
+class UserMailer < ApplicationMailer
+    def login_alert
+    @user=params[:user]
```

Comment:
is instance variable required here?

Reply:
Yes, using the instance variable in app/views/user_mailer/login_alert.html.erb